RELEASES.md

```diff
@@ -197,11 +201,143 @@ To create a minor release:
+run with no load being generated to collect a baseline performance. Once
+baseline is captured, a consistent load is applied across the network. This
+load takes the form of 10% of the running nodes all receiving a consistent
+stream of two hundred transactions per minute each.
+
+During each test net, the following metrics are monitored and collected on each
+node:
+
+* Tendermint rounds per height
+* Maximum connected peers, Minimum connected peers, Rate of change of peer connections
+* Memory resident set size
+* CPU utilization
+* Blocks produced per minute
+* Seconds for each step of consensus (Propose, Prevote, Precommit, Commit)
+* Latency to receive block proposals
+
+For these tests we intentionally target low-powered host machines (with low core
+counts and limited memory) to ensure we observe similar kinds of resource contention 
+and limitation that real-world  deployments of Tendermint experience in production. 
+
+#### 200 Node Testnet
+
+To test the stability and performance of Tendermint in a real world scenario,
+a 200 node test network is run. The network comprises 5 seed nodes, 100
+validators and 95 non-validating full nodes. All nodes begin by dialing
+a subset of the seed nodes to discover peers. The network is run for several
+days, with metrics being collected continuously. In cases of changes to performance
+critical systems, testnets of larger sizes should be considered.
+
+#### Rotating Node Testnet
+
+Real-world deployments of Tendermint frequently see new nodes arrive and old
+nodes exit the network. The rotating node testnet ensures that Tendermint is
+able to handle this reliably. In this test, a network with 10 validators and
+3 seed nodes is started. A rolling set of 25 full nodes are started and each
+connects to the network by dialing one of the seed nodes. Once the node is able
+to blocksync to the head of the chain and begins producing blocks using
+Tendermint consensus it is stopped. Once stopped, a new node is started and
+takes its place. This network is run for several days.
+
+#### Network Partition Testnet
+
+Tendermint is expected to recover from network partitions. A partition where no
+subset of the nodes is left with the super-majority of the stake is expected to
+stop making blocks. Upon alleviation of the partition, the network is expected
+to once again become fully connected and capable of producing blocks. The
+network partition testnet ensures that Tendermint is able to handle this
+reliably at scale. In this test, a network with 100 validators and 95 full
+nodes is started. All validators have equal stake. Once the network is
+producing blocks, a set of firewall rules is deployed to create a partitioned
+network with 50% of the stake on one side and 50% on the other. Once the
+network stops producing blocks, the firewall rules are removed and the nodes
+are monitored to ensure they reconnect and that the network again begins
+producing blocks.
+
+#### Absent Stake Testnet
```

Comment:
I am not sure if I get the point of this scenario. It seems pretty costly, and the actual data or unexpected behavior when can extract from it seems minimal.

I mean, consensus should work with 2/3+1 voting-power. Actually, working with exactly 2/3+1 (2f+1) is the best case for any BFT consensus protocols. The additional n/3 - 1 nodes just spare nodes, which are there for fault-tolerance. If they are offline, consensus will actually be faster than with them being involved.

Reply:
For a node to decide a value, it needs 2f+1 precommits for the value. If there are 3f+1 nodes on the network, then the node has _more_ nodes from which it can receive the votes that it requires. If there are only 2f+1 of the 3f+1 nodes active on the network, then the probability that we observe all of the votes we need to complete consensus decreases. This test ensures that, in the case that voting nodes are offline, the gossip mechanisms are still performant enough to gossip the necessary votes.

Reply:
Ok, so I see this experiment more related to network reliability than consensus. In other words, we use consensus to attest that the overlay network ensures the delivery of messages when a substantial part of the nodes are not active, although (strictly) enough nodes are up.